internal/app/cache/cache.go

```diff
@@ -17,3 +22,116 @@ type Cache interface {
 	// AddWatch adds the watch to the cache and returns whether a stream is open.
 	AddWatch(key string, req envoy_api_v2.DiscoveryRequest) (bool, error)
 }
+
+type cache struct {
+	cache         *ristretto.Cache
+	expireSeconds time.Duration
```

Comment:
may be drop `Seconds` because the type is `time.Duration`?

Reply:
The unit is set to seconds in L60. I'm not sure how else to denote it in terms of seconds to make it clear for the user.

Reply:
`SetWithTTL` takes a `time.Duration`. https://github.com/dgraph-io/ristretto/blob/master/cache.go#L201
I think it doesn't need seconds.